packages/bp/src/core/users/workspace-service.ts

```diff
@@ -63,6 +63,15 @@ export class WorkspaceService {
     return workspaces
   }
 
+  async getAllWorkspaceUsers(): Promise<WorkspaceUser[]> {
+    let allWorkspaceUsers: WorkspaceUser[] = []
+    for (const w of await this.getWorkspaces()) {
+      const users = (await this.getWorkspaceUsers(w.id)) as WorkspaceUser[]
+      allWorkspaceUsers = allWorkspaceUsers.concat(users)
+    }
+    return allWorkspaceUsers
+  }
+
```

Comment:
This method doesn't look useful since we are really just interested in getting single-workspace users..

Reply:
See above